config/deploy.rb

```diff
@@ -25,10 +25,10 @@
 # set :pty, true
 
 # Default value for :linked_files is []
-set :linked_files, %w{config/database.yml config/solr.yml config/default_htaccess_directives}
+set :linked_files, %w{config/database.yml config/solr.yml config/settings/staging.yml}
```

Comment:
Not sure that `config/deploy.rb` should be tied to `settings/staging.yml` in particular.  Maybe that belongs in a subdirectory or another place that is specific to staging?